packages/mobile_app/lib/components/yesterday_poll.dart

```diff
@@ -1,12 +1,17 @@
 import 'package:flutter/material.dart';
+import 'package:int_datetime/int_datetime.dart';
 import 'package:mutable_wrappers/mutable_wrappers.dart';
 
+import 'package:user_poll/bloc.dart';
 import 'package:language/language.dart';
+import 'package:poll_model/poll_model.dart';
+import 'package:userpoll/userpoll.dart';
 import 'magic_poll.dart';
 
 class YesterdayPollSimple extends StatelessWidget {
+  final UserPollBloc userPollBloc;
   final MutableInteger mood;
-  YesterdayPollSimple({@required this.mood});
+  YesterdayPollSimple({@required this.mood, @required this.userPollBloc});
```

Comment:
Optional: Is it the way how it should be -- passing a bloc to a widget and calling its methods? Just out of curiosity

Reply:
It is faster, then use and wait for inherited data.